tests/compile/function/test_pfunc.py

```diff
@@ -1,12 +1,16 @@
 import numpy as np
 import pytest
 
-import aesara
+from aesara import config as aesara_config
```

Comment:
`config` is a variable, not a module, therefore this still imports from the main `__init__`.

Instead:  `from aesara.configdefaults import config`.

Please check the other changed files for this too.
Just realized this is a test file and therefore not critical, but my comment still applies to other files in the codebase.

Reply:
Sounds great, I did that for this case and a couple of other files. 

Before reviewing my edits to see other cases like this module import situation, I am re-running the tests to see if they have good results, right now.